lib/generators/glueby/contract/templates/key_table.rb.erb

```diff
@@ -0,0 +1,14 @@
+class CreateKey < ActiveRecord::Migration<%= migration_version %>
+  def change
+    create_table :keys<%= table_options %> do |t|
+      t.string     :private_key
+      t.string     :public_key
+      t.string     :script_pubkey
+      t.integer    :purpose
+      t.belongs_to :wallet
+      t.timestamps
+    end
+
+    add_index :keys, [:script_pubkey]
```

Comment:
Isn't it better to have a unique constraint on private_key?

Reply:
Added at 8ca322ed89ab1bb551e953f8b6d52cfe68490e4f